pkg/api/users.go

```diff
@@ -167,14 +167,17 @@ func (u *Users) Delete(ctx context.Context, userKey string) error {
 
 // AssignRole assigns a role to a user in your context's environment, by user key, role key and tenant key.
 // Usage Example:
-// `roleAssignment, err := PermitClient.Api.Users.AssignRole(ctx, "user-key", "role-key", "default")`
-func (u *Users) AssignRole(ctx context.Context, userKey string, roleKey string, tenantKey string) (*models.RoleAssignmentRead, error) {
+// `roleAssignment, err := PermitClient.Api.Users.AssignRole(ctx, "user-key", "role-key", "default", "document:mydoc")`
+func (u *Users) AssignRole(ctx context.Context, userKey string, roleKey string, tenantKey string, resourceInstance *string) (*models.RoleAssignmentRead, error) {
```

Comment:
this is actually a breaking change because anyone that uses this function will now have to pass another variable to it

Reply:
create a new function to workaround that - maybe AssignResourceRole ?

Reply:
Oh.. golang... Will do